Versioning.md

```diff
@@ -0,0 +1,13 @@
+# Versioning Enhanced WGSL
+
+(TBD)
+
+As we stabilize for 1.0, we should mark a way for tools (and users) to distinguish our first stable version
+from subsequent releases.
+
+Issues:
+
+* Feature flags or versions?
+* perhaps `version WESL2024`?
+* Does the version need to go in wgsl or just in metadata like wgsl.toml.
```

Comment:
Would prefer for libraries that not every file needs to contain this. Seems an annoyance

Reply:
Yep, I think we'd all agree on that. Having a custom file extension + a version declaration in the main file/the wgsl.toml file should be good. I do really like the Rust concept of editions, so I personally would like to copy that.

Reply:
will mention editions and drop question about wgsl